modules/ossm-config-individual-control-plane-infrastructure-node-console.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-individual-control-plane-infrastructure-node-console_{context}"]
+= Configuring individual control plane components to run on infrastructure nodes using the web console
+
+Perform this task if individual components deployed by the {SMProductShortName} control plane, such as Istiod, the Ingress Gateway, and the Egress Gateway, will run on infrastructure nodes. 
+
+If the control plane will run on a worker node, skip this task.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
+* An account with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Click the {SMProductName} Operator, then click *Istio Service Mesh Control Plane*.
```

Comment:
. Click the *{SMProductName}* Operator and then the *Istio Service Mesh Control Plane* resource.

Reply:
done